### DISCUSSION TITLE
Question about Lora module support.

### DISCUSSION THREAD

**Discussion by EliasA97** [jgromes/RadioLib#215] [Q&A]
Hello, I need to use a Lora module for my project. I found this board: E22-400M22S, that uses the new SX1268 LoRa chip. I wonder if this module is supported by this library.
Link for the LoRa module: http://www.ebyte.com/en/product-view-news.aspx?id=436

Thank's!

**Comment by jgromes** [comment]
From the POV of the library, it's just an SX1268, so it's fully supported. One thing to note is that these modules have an externally controlled RF switch on the TXEN/RXEN pins. RadioLib can control that for you automatically, see the Transmit example: https://github.com/jgromes/RadioLib/blob/master/examples/SX126x/SX126x_Transmit/SX126x_Transmit.ino

**Comment by EliasA97** [comment]
That's great news. Any idea what RXEN/TXEN do? Do I have to use them necessarily or not? Also what are the advantages using them.
Thank's!

**Comment by jgromes** [comment]
RXEN and TXEN control the RF switch - that's what I wrote in my previous message. You have to connect them to any two digital pins on your Arduino, and then provide the pin numbers to the library by calling `radio.setRfSwitchPins(rxen, txen)`, the library will take care of the rest.

**Comment by EliasA97** [comment]
I thought it wasn't necessary to use them. Ok then. I have a last question by the way. I found on the .cpp file a feature to set the LoRa to sleep. What's the command for that?

**Reply by jgromes** [reply]
I'd try `radio.sleep()` - there's a full API documentation here: https://jgromes.github.io/RadioLib/

**Comment by EliasA97** [comment]
Thank you for all your help!!
